doc/adrs/053_add_support_of_custom_anchors_on_frontend.adoc

```diff
@@ -0,0 +1,29 @@
+= ADR-053 - Add support of custom anchors on frontend
+
+== Context
+
+Since https://github.com/eclipse-sirius/sirius-components/issues/565[#565] introducing custom anchors on the backend to support multiple edges between the same elements, we have diverged from Sprotty that use the center of the edge source element and edge target element as anchors to compute the edge route. That difference is one of the causes that makes edges blink when Sprotty renders the diagram, and that something we want to get rid of.
+
+== Decision
+
+We will support custom anchors on the frontend. We will leverage on anchors that has been computed by the backend. This behavior should come before we stop considering the source and target anchors as routing points presented in xref:054_stop_considering_the_source_and_target_anchors_as_routing_points_anymore.adoc[ADR-054].
```

Comment:
Just say in ADR-54 (soon ADR-55), since we tend to move things (here rename ADR), this kind of links can be quite easily broken.

Reply:
Done